multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -60,6 +60,8 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Price (Dollars) to bid for spot instances. Omit for on-demand instances.
 # workerSpotPrice: "0.05"
 
+# Instance type for etcd node
+
```

Comment:
nit: Missing descriptions and examples for newly added parameters

```
#etcdInstanceType: "m3.medium"

# Number of etcd nodes(should be 1, 3, 5 and so on to prevent split-brain)
# etcdCount: 1

# Disk size (GiB) for etcd node
#etcdRootVolumeSize: 30

# Disk size (GiB) for etcd node. This one is user by Etcd to store its key-value data and write-ahead logs
#etcdDataVolumeSize: 30
```